tools/spectral/ipa/rulesets/IPA-105.yaml

```
@@ -29,3 +30,18 @@ rules:
     then:
       field: '@key'
       function: 'eachResourceHasListMethod'
+  xgen-IPA-105-list-method-response-is-get-method-response:
+    description: >-
+      The response body of the List method should consist of the same resource object returned by the Get method. Validation checks that the List method results items reference the same schema as the Get method response.<br/>
```

That is based on assumption that "list operations" return pagginated objects - with results, total etc.
I seen that one before. Would we have validation and helper to check for that assumption  each rule can base on it?
```suggestion
      The response body of the List method should consist of the same resource object returned by the Get method. Validation checks that the List method response contains items property with reference the same schema as the Get method response.<br/>
```

---

@lovisaberggren Yes. I'm aware. My intention for this comment was to add note to the yaml/docs.